src/index.js

```diff
@@ -9,5 +9,58 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Clue from '../src/Clue'
+import '../src/Game'
+import '../src/Player'
+import '../src/Round'
+// import { promises } from 'dns';
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let clue;
+let clueCategories = [];
+let clueInfo = [];
+
+
+
+function categoryFetch() {
+ return fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
+  .then(data => data.json())
+  .then(data => data.data.categories)
+  .then(categories => {
+    let catKeys = Object.keys(categories)
+    catKeys.forEach(key => clueCategories.push({[key]: categories[key]}))
+  })
+  .catch(error => console.log('failure'))
+}
+
+
+function clueFetch() { 
+  return fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
+  .then(data => data.json())
+  .then(data => data.data.clues)
+  .then(clues => {
+    let clueKeys = Object.keys(clues)
+    clueKeys.forEach(key => clueInfo.push(clues[key]))
+  })
+  .catch(error => console.log('failure'))
```

Comment:
What would be helpful for the user in case of this promise failing? Is there some error handling you can do to let them know why things aren't working? Just something to think about for refactoring.

Reply:
You also might want to console.log that error instead of "failure" in case this does fail, you'll have more info on why.